src/coreclr/vm/comutilnative.cpp

```diff
@@ -1851,6 +1851,14 @@ FCIMPL1(MethodTable*, MethodTableNative::InstantiationArg0, MethodTable* mt);
 }
 FCIMPLEND
 
+FCIMPL1(OBJECTHANDLE, MethodTableNative::GetLoaderAllocatorHandle, MethodTable *mt)
```

Comment:
When suspending, the continuation holds the state of the call. The state may need to include loader allocator reference as the method frame could be the only thing holding it alive. This helper is used in related logic.